### DISCUSSION TITLE
[question][@pinia/nuxt] Pinia Nuxt Auto Imports

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [vuejs/pinia#2125] [Help and Questions] [answered]
I'm learning to build Nuxt modules, and checked `@pinia/nuxt` so I can learn from it, and something called my attention;
The autoImports array is empty.  
And I thought "Right! That's why I have to either import `defineStore` manually, or add it in nuxt configuration!".

So now the question is, is this by design? Any downsides of adding `storeToRefs`, `defineStore` to the array by default in a future update?

And also adding the auto imports of the stores in the `~/stores` directory?

Nuxt auto imports composables, layouts, pages, plugins, components, and Vue members by default, I missed it also importing Pinia stores.

Willing to send a PR to contribute.

Wishing you health. 🤝🔥

**Comment by posva** [comment, marked as answer]
It is indeed planned, It's this PR https://github.com/vuejs/pinia/pull/1604 that needs to be fixed and updated
